.github/workflows/docs.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4.4.0
```

Comment:
```suggestion
        uses: actions/setup-python@v4
```

Reply:
The GitHub Actions Marktplace suggest using `v4.4.0`, but I guess `4` will also work 🤷 
<img width="495" alt="Screenshot 2022-12-29 at 17 01 30" src="https://user-images.githubusercontent.com/218003/209978663-7839cddb-4a3a-45cc-87f4-900e24992913.png">

Reply:
Looks like this is an automated widget, which automagically shows the most recent version. On the other hand, the recommended practice is to trust that nothing will break within major release series, and just pin to the major version, like displayed on almost all READMEs of GHA action recipes, like [1].

[1] https://github.com/marketplace/actions/setup-python#basic-usage